CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+## 6.1.0
+
+Teleport 6.1 contains multiple new features, improvements, and bug fixes.
+
+## New Features
+
+### U2F for Kubernetes and SSH sessions
+
+Adds an option to authorize with 2nd factor when connecting to a node/k8s cluster.
+
+[RFD 14](https://github.com/gravitational/teleport/blob/master/rfd/0014-session-2FA.md)
+[RFD 15](https://github.com/gravitational/teleport/blob/master/rfd/0015-2fa-management.md)
```

Comment:
@awly Do you have a short introduction and docs links that would go well in the changelog?

Reply:
Didn't get the time to finish https://github.com/gravitational/teleport/pull/6285 yet.
Once it's done, linking to the docs would be easiest.